operator/images/cluster-setup/content/bin/utils.sh

```diff
@@ -48,6 +48,36 @@ check_deployments() {
   done
 }
 
+check_pod_by_label() {
+  local ns="$1"
+  local label="$2"
+
+  printf -- "- pod with label %s: " "$label"
+  
+  #a loop to check if the pod exists
+  local amountAttepts=20
+  local i=0
+  while [ -z "$(kubectl get pods -l $label -n $ns --no-headers -o custom-columns=':metadata.name')" ]; do
+    printf '.'; sleep 10;
+    i=$((i+1))
+    if [[ $i -eq "${amountAttepts}" ]]; then
+      printf " not found by timeout \n"
+      exit 1
+    fi
+  done
+
+  printf "Exists"
+
+  #a loop to check if the pod is Available and Ready
+  if kubectl wait --for=condition=ready pod -l "$label" -n "$ns" --timeout=100s >/dev/null; then
+    printf ", Ready\n"
+  else
+    kubectl -n "$ns" describe pod -l "$label" 
+    kubectl -n "$ns" logs -l "$label" 
+    exit 1
```

Comment:
Add an error message such as `printf "[ERROR] Pod %s failed to start\n" "$label" >&2`